loolwsd-systemplate-setup

```diff
@@ -55,7 +65,16 @@ find etc/fonts \
 # they link to.
 find $INSTDIR -name 'xpdfimport' |
 while read file; do
-    ldd $file 2>/dev/null
+    if [ ! `uname -s` = "FreeBSD" ]; then
+        ldd $file 2>/dev/null
+    else
+        # lld on FreeBSD prints extraneous line, so chop it
```

Comment:
lld > ldd

Reply:
but why is this necessary? The grep in line 79 won't match the empty line.

Reply:
It is not empty but

    /usr/local/lib/libreoffice/program/xpdfimport:

Reply:
> It is not empty but
> 
> ```
> /usr/local/lib/libreoffice/program/xpdfimport:
> ```

Oh... Would  `cut -s -d " " -f 3` help then?

Reply:
Yep, seems to be working!